backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java

```diff
@@ -81,4 +82,10 @@ public UUID createShop(CreateShopRequest createShopRequest) {
 		Logger.getLogger(InMemoryVirtualQueueRepository.class.getName()).log(Level.SEVERE, "Tried to fetch tokens of a shop which is not in the ACTIVE state.");
 		return new ArrayList<>();
 	}
+
+	public GetShopsByShopOwnerResponse getShopsByShopOwner(String shopOwnerId) {
+		List<Shop> shops = shopMap.entrySet().stream().filter(map -> map.getValue().getShopOwnerId().equals(shopOwnerId))
+				.map(map -> map.getValue()).collect(Collectors.toList());
```

Comment:
Nit. Let's try to reformat these usages like: 
shopMap
    .entrySet()
    .stream()
    .filter(map -> map.getValue().getShopOwnerId().equals(shopOwnerId))
    .map(map -> map.getValue())
    .collect(Collectors.toList())

Reply:
Also do we want to add check here to return only active shops?

Reply:
We are returning all shops for now as we can later give the shopOwner the option to restore the deleted shops or show status of different shops.

Reply:
I meant, let's add comment in the code.

Reply:
Reason being, we'll not be revisiting these comments in the PR at a later stage, but the code we'll be viewing again and again.